robosuite/controllers/sawyer_ik_controller.py

```diff
@@ -71,6 +71,7 @@ def get_control(self, dpos=None, rotation=None):
 
         # Compute new target joint positions if arguments are provided
         if (dpos is not None) and (rotation is not None):
+
```

Comment:
why are the lines of white space added?

Reply:
originally had debugging print statements